packages/x-components/src/x-modules/facets/components/lists/selected-filters-list.vue

```diff
@@ -41,6 +41,7 @@
   import { Component, Prop, Vue } from 'vue-property-decorator';
   import { Facet } from '@empathyco/x-types';
   import { xComponentMixin } from '../../../../components/index';
+  import { toKebabCase } from '../../../../utils';
```

Comment:
Import directly from the file instead of from the barrel. And please, fix also the above `/components/index` import. 
We should try avoid importing from barrels in our code. It is very easy to create cyclical dependencies this way, and even if we don't do so, we are saying that the importer depends on all the files exported by the barrel, which is not true.